src/modules/package.rs

```diff
@@ -174,6 +179,40 @@ mod tests {
             extract_package_version(&package_without_version),
             expected_version
         );
+
```

Comment:
Would it be possible to split this off into its own test? It's a lot easier to see in CI that `test_extract_private_package_version` failed than it is to have to dig through line number/backtrace info to figure out exactly which test in here failed.

Reply:
sure, was just following what was already there, but I'm happy to split them apart :)

Reply:
done 👍